packages/manager/modules/sms/src/sms/batches/history/routing.js

```diff
@@ -39,6 +39,16 @@ export default /* @ngInject */ ($stateProvider) => {
           previousState: 'sms.service.batches.history',
         });
       },
+      goToDeleteBatch: /* @ngInject */ ($state) => (
+        serviceName,
+        batchId,
+        batchName,
+      ) =>
+        $state.go('sms.service.batches.history.delete', {
+          serviceName,
+          batchId,
+          batchName,
+        }),
```

Comment:
I think you can inject serviceName here, this way you wouldn't need a function in the controller.

Reply:
Done!